core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -150,6 +150,15 @@ private[spark] class Executor(
     }
   }
 
+  def killAllTasks (interruptThread: Boolean) : Unit = {
```

Comment:
same here -- say why this is a public thing and how it is used.

also remove the space after killAllTasks